lib/internal/process/execution.js

```diff
@@ -62,33 +62,36 @@ function evalScript(name, body, breakFirstLine, print) {
 
   const asyncESM = require('internal/process/esm_loader');
   const baseUrl = pathToFileURL(module.filename).href;
+  const { loadESM } = asyncESM;
+
+  loadESM(() => {
```

Comment:
So this change means that the ESM loader is initialized now when it wasn’t before, for this code path? What is this code path, exactly? I’d guess `--eval` when the script being run is CommonJS?

Should we do a benchmark to see the performance consequences of this change? @aduh95 

Perhaps another approach would be to define everything you have as this `loadESM` callback as a standalone function, and either run it as an argument to `loadESM` as you have here _if_ `--import` was passed; or just run it directly otherwise. Then maybe the performance hit is isolated to using `--eval` with `--import`, rather than any `--eval` of CommonJS code.

Reply:
correct, see https://github.com/nodejs/node/pull/43942#discussion_r929344190
is there a standard way to perform benchmarking?